PICMI_Python/particles.py

```diff
@@ -27,8 +28,14 @@ class PICMI_Species(_ClassWithInit):
       - particle_shape: Particle shape used for deposition and gather ; if None, the default from the `Simulation` object will be used. Possible values are 'NGP', 'linear', 'quadratic', 'cubic'
     """
 
+    methods_list = ['Boris' , 'Vay', 'Higuera', 'Li']
+    
     def __init__(self, particle_type=None, name=None, charge_state=None, charge=None, mass=None,
```

Comment:
I think the `method=None,` parameter still needs to be added and then the value needs to go into a `self.method = method`?

Reply:
Yes, that is correct, method needs to be added to the parameter list.